Modules/Devices/src/Devices.Application/Tiers/Queries/ListTiers/Handler.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Application.Tiers.DTOs;
+using MediatR;
+
+namespace Backbone.Modules.Devices.Application.Tiers.Queries.ListTiers;
+public class Handler : IRequestHandler<ListTiersQuery, ListTiersResponse>
+{
+    private readonly ITierRepository _tierRepository;
+
+    public Handler(ITierRepository repository)
+    {
+        _tierRepository = repository;
+    }
+
+    public async Task<ListTiersResponse> Handle(ListTiersQuery request, CancellationToken cancellationToken)
+    {
+        var dbPaginationResult = await _tierRepository.FindAll(request.PaginationFilter);
+        var tiersDTOList = dbPaginationResult.ItemsOnPage.Select(el =>
+        {
+            return new TierDTO(el.Id, el.Name);
+        }).ToList();
```

Comment:
You should always prefer the short form:
```suggestion
        var tiersDTOList = dbPaginationResult.ItemsOnPage.Select(el => new TierDTO(el.Id, el.Name)).ToList();
```